### PR TITLE
release(v1.5.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v1.5.0-alpha.0 release.

### CODE REVIEW

**Comment by smira** [comment]
I disabled Azure image upload as it wasn't tested, and I don't want it to break the release process.

File: website/content/v1.5/reference/cli.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ talosctl cluster create [flags]
       --bad-rtc                                  launch VM with bad RTC state (QEMU only)
       --cidr string                              CIDR of the cluster network (IPv4, ULA network for IPv6 is derived in automated way) (default "10.5.0.0/24")
       --cni-bin-path strings                     search path for CNI binaries (VM only) (default [/home/user/.talos/cni/bin])
-      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.4.0-alpha.4/talosctl-cni-bundle-${ARCH}.tar.gz")
+      --cni-bundle-url string                    URL to download CNI bundle from (VM only) (default "https://github.com/siderolabs/talos/releases/download/v1.5.0-alpha.0-dirty/talosctl-cni-bundle-${ARCH}.tar.gz")
```
Comment (rsmitty) [inline website/content/v1.5/reference/cli.md:99]: needs a make docs?

**Review by frezbo — APPROVED** [review]

File: website/content/v1.5/reference/cli.md (hunk 1 above)
Comment (smira) [reply website/content/v1.5/reference/cli.md:99]: yeah, yet another one... looks like

**Review by Unix4ever — APPROVED** [review]

**Comment by smira** [comment]
extensions build is broken... but now I'm not sure how it got through CI (?). I have a fix in the release PR for extensions

**Pull request merged**